package.json

```diff
@@ -61,7 +61,6 @@
         "lerna": "^3.20.2",
         "npm-run-all": "^4.1.5",
         "patch-package": "^6.2.1",
-        "postinstall-postinstall": "^2.0.0",
```

Comment:
Probably worth reverting that out of precaution now that the esbuild issue has been addressed (#3087)

Reply:
yeah, that was just experiments